plugins/scaffolder-backend/package.json

```diff
@@ -93,6 +93,7 @@
     "concat-stream": "^2.0.0",
     "express": "^4.17.1",
     "express-promise-router": "^4.1.0",
+    "fast-glob": "^3.3.3",
```

Comment:
Hm, we use `"minimatch": "^9.0.0"` in all other places where globbing is needed - could you use that one for consistency?

Reply:
Doing this with `minimatch` would be inefficient, as you'd need to list all the files in the directory and loop through them. Instead I've switched to using `globby` which is a wrapper around `fast-glob`. `globby` is already used by some of the actions in this plugin, so no new dependency.

Reply:
Oh yeah sorry, globby of course. Thanks